cmd/sprint-automation/main.go

```diff
@@ -175,6 +175,8 @@ const (
 	primaryOnCallQuery     = "DPTP Primary On-Call"
 	secondaryUSOnCallQuery = "DPTP Secondary On-Call (US)"
 	secondaryEUOnCallQuery = "DPTP Secondary On-Call (EU)"
+	helpdeskQuery          = "DPTP Help Desk"
+	intakeQuery            = "DPTP Intake"
```

Comment:
This should not be here, looks like this some kind of `git commit --amend` that went wrong. Please clean up the PR so it does not contain unrelated changes.

Reply:
/hold